internal/handlers/publishers.go

```diff
@@ -58,14 +58,20 @@ func (p *Publisher) GetPublisher(ctx *fiber.Ctx) error {
 
 // PostPublisher creates a new publisher.
 func (p *Publisher) PostPublisher(ctx *fiber.Ctx) error {
-	publisher := new(requests.Publisher)
+	publisherReq := requests.Publisher{}
 
-	if err := ctx.BodyParser(&publisher); err != nil {
-		return common.Error(fiber.StatusBadRequest, "can't create Publisher", "invalid json")
+	if err := ctx.BodyParser(&publisherReq); err != nil {
+		return common.Error(fiber.StatusBadRequest, "can't create Publisher", "invalid json", err)
```

Comment:
I'm not sure about this one, I think `err` should go into `detail`.

Reply:
We can discuss about that tomorrow 👌